deployments/ocis-office/helmfile.yaml

```diff
@@ -63,6 +63,8 @@ releases:
           enabled: true
       - ingress:
           enabled: true
+          annotations:
+            nginx.org/websocket-services: documentserver
```

Comment:
Please note that this is for https://github.com/nginxinc/kubernetes-ingress and not for https://github.com/kubernetes/ingress-nginx

Is this intended?

Reply:
Well it worked with ingress-nginx installed by:

```
helm install nginx-ingress nginx-stable/nginx-ingress
```

Reply:
what's the repo url for your `nginx-stable` repo?

Reply:
Url was https://helm.nginx.com/stable

Got it now. Will test with the other one.

Reply:
https://kubernetes.github.io/ingress-nginx/user-guide/miscellaneous/#websockets is relevant in this case